### PR TITLE
[OUDS] feat: Add "Elevation" tokens, utilities and documentation

### LINKED ISSUE #2596: [OUDS] Tokens: Elevation
## Description

The aim of this issue is to study and implement the elevation tokens taking into account the cohesive multi-platform approach, the customization by libraries inheriting OUDS Web, but also what we're using for OUDS Web: Boosted, Bootstrap etc.
- Define the possibilities
- Define the constraints
- Define the impacts
- Implement the tokens (with a future Style Dictionary usage in mind)

Reminder: there will be in the end 3 layers of tokens:
- **Raw primitive values: Insertion of brand foundations.**
  - These tokens will be customizable by libraries inheriting OUDS Web.
  - These tokens cannot be used by developers using OUDS Web.
- **Semantic applications: Depends on raw primitive values.**
  - Mapping between these tokens and the raw primitive values will be customizable by libraries inheriting OUDS Web.
  - These tokens can be used by developers using OUDS Web.
- **Component applications: Depends on semantic applications.**:
  - Mapping between these tokens and the semantic tokens should rarely be customized by libraries inheriting OUDS Web.
  - These tokens can be used by developers using OUDS Web.

#### X

|  Token name | Token value |
|---|---|
| elevation-x-0 | 0 |

#### Y

|  Token name | Token value |
|---|---|
| elevation-y-0 | 0 |
| elevation-y-100 | 1 |
| elevation-y-200 | 2 |
| elevation-y-300 | 4 |
| elevation-y-400 | 8 |
| elevation-y-500 | 12 |
| elevation-y-600 | 20 |

#### Blur

|  Token name | Token value |
|---|---|
| elevation-blur-0 | 0 |
| elevation-blur-100 | 1 |
| elevation-blur-200 | 2 |
| elevation-blur-300 | 3 |
| elevation-blur-400 | 4 |
| elevation-blur-500 | 8 |
| elevation-blur-600 | 12 |
| elevation-blur-700 | 20 |

#### Spread

|  Token name | Token value |
|---|---|
| elevation-spread-n100 | -1 |
| elevation-spread-n200 | -2 |
| elevation-spread-n300 | -4 |
| elevation-spread-n400 | -8 |
| elevation-spread-0 | 0 |
| elevation-spread-300 | 3 |

#### Box shadow (Composite)

> [!CAUTION]
> Figma doesn't accept composite tokens

|  Token name | Token value |
|---|---|
| elevation-bottom-0 | box-shadow: 0 0px 0px 0 rgba({core.color.transparent.black.0}); |
| elevation-bottom-1-100 | box-shadow: 0 1px 2px 0 rgba({core.color.transparent.black.100}); |
| elevation-bottom-1-200 | box-shadow: 0 1px 2px 0 rgba({core.color.transparent.black.200}); |
| elevation-bottom-1-300 | box-shadow: 0 1px 2px 0 rgba({core.color.transparent.black.300}); |
| elevation-bottom-1-400 | box-shadow: 0 1px 2px 0 rgba({core.color.transparent.black.400}); |
| elevation-bottom-1-500 | box-shadow: 0 1px 2px 0 rgba({core.color.transparent.black.500}); |
| elevation-bottom-1-600 | box-shadow: 0 1px 2px 0 rgba({core.color.transparent.black.600}); |
| elevation-bottom-2-100 | box-shadow: 0 2px 3px 0 rgba({core.color.transparent.black.100}); |
| elevation-bottom-2-200 | box-shadow: 0 2px 3px 0 rgba({core.color.transparent.black.200}); |
| elevation-bottom-2-300 | box-shadow: 0 2px 3px 0 rgba({core.color.transparent.black.300}); |
| elevation-bottom-2-400 | box-shadow: 0 2px 3px 0 rgba({core.color.transparent.black.400}); |
| elevation-bottom-2-500 | box-shadow: 0 2px 3px 0 rgba({core.color.transparent.black.500}); |
| elevation-bottom-2-600 | box-shadow: 0 2px 3px 0 rgba({core.color.transparent.black.600}); |
| elevation-bottom-3-100 | box-shadow: 0 4px 4px -1px rgba({core.color.transparent.black.100}); |
| elevation-bottom-3-200 | box-shadow: 0 4px 4px -1px rgba({core.color.transparent.black.200}); |
| elevation-bottom-3-300 | box-shadow: 0 4px 4px -1px rgba({core.color.transparent.black.300}); |
| elevation-bottom-3-400 | box-shadow: 0 4px 4px -1px rgba({core.color.transparent.black.400}); |
| elevation-bottom-3-500 | box-shadow: 0 4px 4px -1px rgba({core.color.transparent.black.500}); |
| elevation-bottom-3-600 | box-shadow: 0 4px 4px -1px rgba({core.color.transparent.black.600}); |
| elevation-bottom-4-100 | box-shadow: 0 8px 8px -2px rgba({core.color.transparent.black.100}); |
| elevation-bottom-4-200 | box-shadow: 0 8px 8px -2px rgba({core.color.transparent.black.200}); |
| elevation-bottom-4-300 | box-shadow: 0 8px 8px -2px rgba({core.color.transparent.black.300}); |
| elevation-bottom-4-400 | box-shadow: 0 8px 8px -2px rgba({core.color.transparent.black.400}); |
| elevation-bottom-4-500 | box-shadow: 0 8px 8px -2px rgba({core.color.transparent.black.500}); |
| elevation-bottom-4-600 | box-shadow: 0 8px 8px -2px rgba({core.color.transparent.black.600}); |
| elevation-bottom-5-100 | box-shadow: 0 12px 12px -4px rgba({core.color.transparent.black.100}); |
| elevation-bottom-5-200 | box-shadow: 0 12px 12px -4px rgba({core.color.transparent.black.200}); |
| elevation-bottom-5-300 | box-shadow: 0 12px 12px -4px rgba({core.color.transparent.black.300}); |
| elevation-bottom-5-400 | box-shadow: 0 12px 12px -4px rgba({core.color.transparent.black.400}); |
| elevation-bottom-5-500 | box-shadow: 0 12px 12px -4px rgba({core.color.transparent.black.500}); |
| elevation-bottom-5-600 | box-shadow: 0 12px 12px -4px rgba({core.color.transparent.black.600}); |
| elevation-bottom-6-100 | box-shadow: 0 20px 20px -8px rgba({core.color.transparent.black.100}); |
| elevation-bottom-6-200 | box-shadow: 0 20px 20px -8px rgba({core.color.transparent.black.200}); |
| elevation-bottom-6-300 | box-shadow: 0 20px 20px -8px rgba({core.color.transparent.black.300}); |
| elevation-bottom-6-400 | box-shadow: 0 20px 20px -8px rgba({core.color.transparent.black.400}); |
| elevation-bottom-6-500 | box-shadow: 0 20px 20px -8px rgba({core.color.transparent.black.500}); |
| elevation-bottom-6-600 | box-shadow: 0 20px 20px -8px rgba({core.color.transparent.black.600}); |

#### X

|  Token name | Token value |
|---|---|
| elevation-x-none | elevation-x-0 |
| elevation-x-raised | elevation-x-0 |
| elevation-x-drag | elevation-x-0 |
| elevation-x-overlay-default | elevation-x-0 |
| elevation-x-overlay-emphasized | elevation-x-0 |
| elevation-x-sticky-default | elevation-x-0 |
| elevation-x-sticky-emphasized | elevation-x-0 |
| elevation-x-sticky-navigation-scrolled | elevation-x-0 |
| elevation-x-focus | elevation-x-0 |

#### Y

|  Token name | Token value |
|---|---|
| elevation-y-none | elevation-y-0 |
| elevation-y-raised | elevation-y-100 |
| elevation-y-drag | elevation-y-300 |
| elevation-y-overlay-default | elevation-y-200 |
| elevation-y-overlay-emphasized | elevation-y-500 |
| elevation-y-sticky-default | elevation-y-300 |
| elevation-y-sticky-emphasized | elevation-y-300 |
| elevation-y-sticky-navigation-scrolled | elevation-y-300 |
| elevation-y-focus | elevation-y-0 |

#### Blur

|  Token name | Token value |
|---|---|
| elevation-blur-none | elevation-blur-0 |
| elevation-blur-raised | elevation-blur-200 |
| elevation-blur-drag | elevation-blur-400 |
| elevation-blur-overlay-default | elevation-blur-300 |
| elevation-blur-overlay-emphasized | elevation-blur-600 |
| elevation-blur-sticky-default | elevation-blur-400 |
| elevation-blur-sticky-emphasized | elevation-blur-400 |
| elevation-blur-sticky-navigation-scrolled | elevation-blur-400 |
| elevation-blur-focus | elevation-blur-0 |

#### Spread

|  Token name | Token value |
|---|---|
| elevation-spread-none | elevation-spread-0 |
| elevation-spread-raised | elevation-spread-0 |
| elevation-spread-drag | elevation-spread-n100 |
| elevation-spread-overlay-default | elevation-spread-n100 |
| elevation-spread-overlay-emphasized | elevation-spread-n300 |
| elevation-spread-sticky-default | elevation-spread-n100 |
| elevation-spread-sticky-emphasized | elevation-spread-n100 |
| elevation-spread-sticky-navigation-scrolled | elevation-spread-n100 |
| elevation-spread-focus | elevation-spread-300 |

#### Color

|  Token name | Token value (light) | Token value (inverse) | Token value (dark) |
|---|---|---|---|
| elevation-color-none | color-transparent-black-0 | color-transparent-black-0 | color-transparent-black-0 |
| elevation-color-raised | color-transparent-black-500 | color-transparent-black-500 | color-transparent-black-500 |
| elevation-color-drag | color-transparent-black-600 | color-transparent-black-600 | color-transparent-black-600 |
| elevation-color-overlay-default | color-transparent-black-400 | color-transparent-black-400 | color-transparent-black-400 |
| elevation-color-overlay-emphasized | color-transparent-black-300 | color-transparent-black-300 | color-transparent-black-300 |
| elevation-color-sticky-default | color-transparent-black-300 | color-transparent-black-300 | color-transparent-black-300 |
| elevation-color-sticky-emphasized | color-transparent-black-300 | color-transparent-black-300 | color-transparent-black-300 |
| elevation-color-sticky-navigation-scrolled | color-transparent-black-300 | color-transparent-black-300 | color-transparent-black-300 |
| elevation-color-focus | color-transparent-white-900 | color-transparent-white-900 | color-transparent-white-900 |

#### Box shadow (Composite)

> [!CAUTION]
> Figma doesn't accept composite tokens

|  Token name | Token value (light) | Token value (inverse) | Token value (dark) |
|---|---|---|---|
| elevation-none | elevation-bottom-0 | elevation-bottom-0 | elevation-bottom-0 |
| elevation-raised | elevation-bottom-1-500 | elevation-bottom-1-500 | elevation-bottom-1-500 |
| elevation-drag | elevation-bottom-3-500 | elevation-bottom-3-500 | elevation-bottom-3-500 |
| elevation-overlay-default | elevation-bottom-2-400 | elevation-bottom-2-400 | elevation-bottom-2-400 |
| elevation-overlay-emphasized | elevation-bottom-5-300 | elevation-bottom-5-300 | elevation-bottom-5-300 |
| elevation-sticky-default | elevation-bottom-3-300 | elevation-bottom-3-300 | elevation-bottom-3-300 |
| elevation-sticky-emphasized | elevation-bottom-3-300 | elevation-bottom-3-300 | elevation-bottom-3-300 |
| elevation-sticky-navigation-scrolled | elevation-bottom-3-300 | elevation-bottom-3-300 | elevation-bottom-3-300 |
| elevation-focus | box-shadow: 0 0px 0px 3px rgba({sys-color-transparent-black-900}); | box-shadow: 0 0px 0px 3px rgba({sys-color-transparent-white-900}); | box-shadow: 0 0px 0px 3px rgba({sys-color-transparent-white-900}); |

---

## Study

- [x] Study the architecture of these tokens, and define the technical details

### Technical details

TODO

### PR DESCRIPTION
**Pull request by louismaximepiton**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Related issues

Closes #2596.
Listed in #2589.

### Description

Introducing all the Elevation tokens, `box-shadow` tokens. Introducing their utilities and documentation as well.

|  Branch  |  `ouds/main`  |  `ouds/main-lmp-elevation-token`  |  Inflate/deflate  |
|---|---|---|---|
|  `ouds-web.css`  |  360 607  |  360 477  |  -0% (-130 B)  |
|  `ouds-web-bootstrap.css`  |  360 835  |  361 140  |  +0.1% (+305 B)  |
|  `ouds-web-grid.css`  |  73 461  |  73 462  |  +0% (+1 B)  |
|  `ouds-web-reboot.css`  |  19 357  |  19 262  |  -1% (-95 B)  |
|  `ouds-web-utilities.css`  |  117 910  |  117 780  |  -0.1% (-130 B)  |

### Enrich the documentation

This PR adds the "Utilities > Shadows" page based on:
- the previous [corresponding Boosted page](https://boosted.orange.com/docs/5.3/utilities/shadows/) (see [code source](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/utilities/shadows.md))
- the [corresponding Bootstrap page](https://getbootstrap.com/docs/5.3/utilities/shadows/) (see [code source](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/utilities/shadows.md))

To be added in #2589:
- [ ] Uncomment elements whenever StyleDictionary is used for generating the tokens.

This PR adds the "Layout > Z-index" page based on:
- the previous [corresponding Boosted page](https://boosted.orange.com/docs/5.3/layout/zindex/) (see [code source](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/layout/zindex.md))
- the [corresponding Bootstrap page](https://getbootstrap.com/docs/5.3/layout/zindex/) (see [code source](https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.3/layout/zindex.md))

To be added in #2589:
- [ ] Uncomment elements whenever Navigation elements are developed
- [ ] Uncomment elements whenever Tooltip elements are developed
- [ ] Uncomment elements whenever Popover elements are developed
- [ ] Uncomment elements whenever Modal elements are developed
- [ ] Uncomment elements whenever Dropdown elements are developed
- [ ] Uncomment elements whenever a component element is developed

### Types of change

- New feature (non-breaking change which adds functionality)

### Live previews

- <https://deploy-preview-2710--boosted.netlify.app/docs/layout/zindex>
- <https://deploy-preview-2710--boosted.netlify.app/docs/utilities/shadows>
- <https://deploy-preview-2710--boosted.netlify.app/docs/migration>
- <https://deploy-preview-2710--boosted.netlify.app/docs/migration-from-boosted>

#### Contribution

- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

#### Accessibility

- [x] My change follows accessibility good practices; I have at least run axe

#### Design

- [x] My change respects the design guidelines defined in [Orange Design System](https://oran.ge/dsweb)
- [x] My change is compatible with a responsive display

#### Development

- [x] My change follows the [developer guide](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/wiki/Developer-guide)
- [x] I have added JavaScript unit tests to cover my changes
- [x] I have added SCSS unit tests to cover my changes

#### Documentation

- [x] My change introduces changes to the documentation and/or I have updated the documentation accordingly

### Checklist (for Core Team only)

- [x] My change introduces changes to the migration guide
- [x] My new component is well displayed in [Storybook](https://deploy-preview-2710--boosted.netlify.app/storybook)
- [x] My new component is compatible with RTL
- [x] Manually run [BrowserStack tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/browserstack.yml)
- [ ] Manually test browser compatibility with BrowserStack (Chrome >= 60, Firefox >= 60 (+ ESR), Edge, Safari >= 12, iOS Safari, Chrome & Firefox on Android)
- [ ] Code review
- [ ] Design review
- [ ] A11y review

#### After the merge

- [ ] Manually launch [Percy tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/percy.yml)

### CODE REVIEW

File: scss/tokens/_raw.scss [hunk 1]
```diff
@@ -17,3 +17,130 @@ $ouds-opacity-700: .64 !default;
 // $ouds-opacity-800: .88 !default;
 $ouds-opacity-900: 1 !default;
 // scss-docs-end ouds-opacities-variables
+
+// Elevation
+
+// scss-docs-start ouds-raw-zindex
```
Comment (julien-deramond) [inline scss/tokens/_raw.scss:23]: These tokens won't be generated. We can keep raw values directly in `_variables.scss` (or anywhere else).

File: scss/_variables.scss [hunk 2]
```diff
@@ -648,10 +648,30 @@ $focus-visible-outer-offset:          $focus-visible-inner-width !default; // OU
 $focus-visible-outer-color:           $black !default; // OUDS mod
 // scss-docs-end focus-visible-variables
 
+// fusv-disable
+// scss-docs-start box-shadow-variables-ouds
```
Comment (louismaximepiton) [inline scss/_variables.scss:652]: wondering if this part should be inside the semantic file or variables file.

File: scss/tokens/_raw.scss [hunk 3]
```diff
@@ -17,3 +17,34 @@ $ouds-opacity-700: .64 !default;
 // $ouds-opacity-800: .80 !default;
 $ouds-opacity-900: 1 !default;
 // scss-docs-end ouds-opacities-variables
+
+// Elevation
+
+// scss-docs-start ouds-raw-shadow
+// Box-shadow
+$ouds-box-shadow-x-0:           0 !default;
```
Comment (julien-deramond) [inline scss/tokens/_raw.scss:25]: For these tokens, the "elevation" word is missing.

File: scss/tokens/_semantic.scss [hunk 4]
```diff
@@ -13,3 +13,58 @@ $ouds-opacities: (
   "opaque":      $ouds-opacity-900
 ) !default;
 // scss-docs-end ouds-opacities-variables-maps
+
+// Elevation
+
+// scss-docs-start ouds-sem-shadow
+$ouds-box-shadow-x-none:                      $ouds-box-shadow-x-0 !default;
```
Comment (julien-deramond) [inline scss/tokens/_semantic.scss:20]: For these tokens, the "elevation" word is missing

**Comment by julien-deramond** [comment]
TODO in this PR after some modifications in the corresponding issue:
- [ ] `elevation-box-shadow-*` ‭→ `elevation-*`
- [ ] `elevation-*-emphasis` ‭→ `elevation-*-emphasized`
- [ ] new `elevation-bottom-0` value ‭→ `box-shadow: 0 0px 0px 0 rgba({core.color.transparent.black.0});`
- [ ] new `elevation-drag` value ‭→ `elevation-bottom-3-500`

File: scss/_variables.scss (hunk 2 above)
Comment (julien-deramond) [reply scss/_variables.scss:652]: Maybe `box-shadow-variables-ouds` should rather be `elevation-variables-ouds`

File: scss/_variables.scss [hunk 5]
```diff
@@ -1406,6 +1426,9 @@ $zindex-dropdown:                   1000 !default;
 $zindex-sticky:                     1020 !default;
 $zindex-fixed:                      1030 !default;
 $zindex-back-to-top:                1035 !default; // OUDS mod
+// fusv-disable
+$zindex-spinner:                    1038 !default; // OUDS mod
```
Comment (julien-deramond) [inline scss/_variables.scss:1430]: Let's drop it, as it's not useful for now. And it's not a token that we will receive from Figma.

File: scss/tokens/_raw.scss [hunk 6]
```diff
@@ -17,3 +17,34 @@ $ouds-opacity-700: .64 !default;
 // $ouds-opacity-800: .80 !default;
 $ouds-opacity-900: 1 !default;
 // scss-docs-end ouds-opacities-variables
+
+// Elevation
+
+// scss-docs-start ouds-raw-shadow
+// Box-shadow
+$ouds-elevation-x-0:           0 !default;
```
Comment (julien-deramond) [inline scss/tokens/_raw.scss:25]: Waiting for more information, but there could be also a `$ouds-elevation-x-200`.

File: scss/tokens/_raw.scss [hunk 7]
```diff
@@ -17,3 +17,34 @@ $ouds-opacity-700: .64 !default;
 // $ouds-opacity-800: .80 !default;
 $ouds-opacity-900: 1 !default;
 // scss-docs-end ouds-opacities-variables
+
+// Elevation
+
+// scss-docs-start ouds-raw-shadow
+// Box-shadow
```
Comment (julien-deramond) [inline scss/tokens/_raw.scss:24]: This comment can be removed.

File: scss/tokens/_raw.scss [hunk 8]
```diff
@@ -17,3 +17,34 @@ $ouds-opacity-700: .64 !default;
...
+$ouds-elevation-blur-600:      12px !default;
+// $ouds-elevation-blur-700:      20px !default;
+
+// $ouds-elevation-spread-n400:   -8px !default;
+$ouds-elevation-spread-n300:   -4px !default;
+// $ouds-elevation-spread-n200:   -2px !default;
+$ouds-elevation-spread-n100:   -1px !default;
+$ouds-elevation-spread-0:      0 !default;
+$ouds-elevation-spread-300:    3px !default;
+// scss-docs-end ouds-raw-shadow
```
Comment (julien-deramond) [inline scss/tokens/_raw.scss:50]: Not 100% sure yet, but we could use composite tokens too (if technically possible), something like:
`$elevation-bottom-0: $ouds-elevation-x-none $ouds-elevation-y-none $ouds-elevation-blur-none $ouds-elevation-spread-none $ouds-elevation-color-none` to define `$elevation-none: $elevation-bottom-0` 🤷

File: scss/tokens/_semantic.scss [hunk 9]
```diff
@@ -13,3 +13,58 @@ $ouds-opacities: (
...
+
+// scss-docs-start ouds-sem-shadow
+$ouds-elevation-x-none:                      $ouds-elevation-x-0 !default;
+$ouds-elevation-x-raised:                    $ouds-elevation-x-0 !default;
+$ouds-elevation-x-drag:                      $ouds-elevation-x-0 !default;
+$ouds-elevation-x-overlay-default:           $ouds-elevation-x-0 !default;
+$ouds-elevation-x-overlay-emphasized:        $ouds-elevation-x-0 !default;
+$ouds-elevation-x-sticky-default:            $ouds-elevation-x-0 !default;
+$ouds-elevation-x-emphasized:                $ouds-elevation-x-0 !default;
+$ouds-elevation-x-navigation-scrolled:       $ouds-elevation-x-0 !default;
```
Comment (julien-deramond) [inline scss/tokens/_semantic.scss:27]: `$ouds-elevation-x-navigation-scrolled` doesn't exist and should be `$ouds-elevation-x-sticky-navigation-scrolled`

File: scss/tokens/_semantic.scss [hunk 10]
```diff
@@ -13,3 +13,58 @@ $ouds-opacities: (
...
+// TODO: Check the colors when the color tokens will be added.
+$ouds-elevation-color-none:                  rgba(0, 0, 0, 0) !default;
+$ouds-elevation-color-raised:                rgba(0, 0, 0, .32) !default;
+$ouds-elevation-color-drag:                  rgba(0, 0, 0, .48) !default;
+$ouds-elevation-color-overlay-default:       rgba(0, 0, 0, .24) !default;
+$ouds-elevation-color-overlay-emphasized:    rgba(0, 0, 0, .16) !default;
+$ouds-elevation-color-sticky-default:        rgba(0, 0, 0, .16) !default;
+$ouds-elevation-color-emphasized:            rgba(0, 0, 0, .16) !default;
+$ouds-elevation-color-navigation-scrolled:   rgba(0, 0, 0, .16) !default;
+$ouds-elevation-color-focus:                 rgba(0, 0, 0, 1) !default;
```
Comment (julien-deramond) [inline scss/tokens/_semantic.scss:69]: Note: colors have changed. Let's check if it has an impact.

File: scss/tokens/_raw.scss [hunk 11]
```diff
@@ -17,3 +17,34 @@ $ouds-opacity-700: .64 !default;
 // $ouds-opacity-800: .80 !default;
 $ouds-opacity-900: 1 !default;
 // scss-docs-end ouds-opacities-variables
+
+// Elevation
+
+// scss-docs-start ouds-raw-shadow
```
Comment (julien-deramond) [inline scss/tokens/_raw.scss:23]: Should probably be `ouds-raw-elevation`. At least replacing "shadow" by "elevation".

File: scss/tokens/_semantic.scss (hunk 9 above)
Comment (julien-deramond) [reply scss/tokens/_semantic.scss]: Fixed by https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2710/commits/70df90752fb68034339d3de8f134533f2d0c14a4

File: scss/tokens/_semantic.scss [hunk 12]
```diff
@@ -13,3 +13,58 @@ $ouds-opacities: (
...
+// Elevation
+
+// scss-docs-start ouds-sem-shadow
+$ouds-elevation-x-none:                      $ouds-elevation-x-0 !default;
+$ouds-elevation-x-raised:                    $ouds-elevation-x-0 !default;
+$ouds-elevation-x-drag:                      $ouds-elevation-x-0 !default;
+$ouds-elevation-x-overlay-default:           $ouds-elevation-x-0 !default;
+$ouds-elevation-x-overlay-emphasized:        $ouds-elevation-x-0 !default;
+$ouds-elevation-x-sticky-default:            $ouds-elevation-x-0 !default;
+$ouds-elevation-x-emphasized:                $ouds-elevation-x-0 !default;
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply scss/tokens/_semantic.scss]: Fixed by https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2710/commits/70df90752fb68034339d3de8f134533f2d0c14a4

**Comment by julien-deramond** [comment]
Closing this PR as superseded by #2741

**Pull request closed without merging**